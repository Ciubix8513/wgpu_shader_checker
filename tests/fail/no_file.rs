use wgpu_shader_checker::include_wgsl;

fn main() {
    include_wgsl!("./shaders/meowwow.wgsl")
}
