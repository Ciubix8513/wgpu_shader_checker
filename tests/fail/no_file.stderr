error: No such file or directory
 --> tests/fail/no_file.rs:4:5
  |
4 |     include_wgsl!("./shaders/meowwow.wgsl")
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `include_wgsl` (in Nightly builds, run with -Z macro-backtrace for more info)
